apps/demo-pkp-social-auth-next-ts/package.json

```diff
@@ -9,26 +9,30 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@lit-protocol/auth-helpers": "file:../../packages/auth-helpers",
-    "@lit-protocol/constants": "file:../../packages/constants",
-    "@lit-protocol/lit-auth-client": "file:../../packages/lit-auth-client",
-    "@lit-protocol/pkp-ethers": "file:../../packages/pkp-ethers",
-    "@lit-protocol/types": "file:../../packages/types",
+    "@lit-protocol/auth-helpers": "^3.0.5",
+    "@lit-protocol/constants": "^3.0.5",
+    "@lit-protocol/lit-auth-client": "^3.0.5",
+    "@lit-protocol/pkp-ethers": "^3.0.5",
+    "@lit-protocol/types": "^3.0.5",
     "@radix-ui/react-radio-group": "^1.1.3",
     "@stytch/nextjs": "^9.0.0",
+    "@stytch/vanilla-js": "^3.0.1",
```

Comment:
seems to have been accidentally introduced. remove the `@stytch/vanilla-js` package